analysis/deep2_r23_o32/general.py

```diff
@@ -339,20 +401,26 @@ def run_individual_functions(fitspath, want, dataset='n_Bins', n_split=3, adapti
             galinbin = [400, 400, 400, 400, 400, 400, 409]
         bin_pdf_pages = join(fitspath, name_dict['gridpdf_suffix'])
         bin_outfile = join(fitspath, name_dict['gridnpz_suffix'])
-        n_bins_grid_analysis.n_times_binned(fitspath, bin_pdf_pages, bin_outfile, n_split,
-                                            individual_ID, R23, O32, SNR3, data3, galinbin)
+        n_bins_grid_analysis.n_times_binned(fitspath, bin_pdf_pages,
+                                            bin_outfile, n_split,
+                                            individual_ID, R23, O32, SNR3,
+                                            data3, galinbin, log=log)
         # Starting Stacking
-        Stack_name = 'Stacking_Masked_MasterGrid_' + dataset + '.pdf'
+        if mask:
+            stack_name = dataset + name_dict['Stackname']
+        else:
+            stack_name = dataset + name_dict['Stackname_nomask']
         stackboth_mastergrid.master_stacking(fitspath, fitspath_ini, dataset,
-                                             bin_outfile, Stack_name, mask=mask)
+                                             bin_outfile, stack_name,
+                                             mask=mask, log=log)
 
     if want == 'zoom':
-        Stack_name = 'Stacking_Masked_MasterGrid_' + dataset + '.fits'
-        outfile_grid = join(fitspath, Stack_name)
+        outfile_grid = join(fitspath, filename_dict['comp_spec'])
```

Comment:
Should this name, filename_dict['comp_spec'], live in the Zcalbase_gal name_dict? I am not sure, but I think only Zcalbase_gal uses this name.

Reply:
This standardizes so that all composite spectra are in a FITS file called 'composite_spectra.fits'